ci/autoscaler/pipeline.yml

```diff
@@ -298,25 +292,26 @@ jobs:
   - in_parallel:
     - get: bbl-state
       passed: [deploy-cf]
-    - get: app-autoscaler-release-main
-      passed: [unit-tests-main, integration-tests-main]
+    - get: app-autoscaler-release-ci
   - task: run-pre-release-script
-    file: app-autoscaler-release-main/ci/autoscaler/tasks/run-prerelease-script.yml
+    file: app-autoscaler-release-ci/ci/autoscaler/tasks/run-prerelease-script.yml
     params:
       SCRIPT_NAME: ./scripts/update
     input_mapping:
-      release: app-autoscaler-release-main
+      release: app-autoscaler-release-ci
     output_mapping:
-      release: app-autoscaler-release-main
+      release: app-autoscaler-release-ci
   - task: deploy-autoscaler
-    file: app-autoscaler-release-main/ci/autoscaler/tasks/deploy-autoscaler.yml
+    file: app-autoscaler-release-ci/ci/autoscaler/tasks/deploy-autoscaler.yml
     params:
       SYSTEM_DOMAIN: autoscaler.ci.cloudfoundry.org
       <<: *app-autoscaler-ops-files
     input_mapping:
-      app-autoscaler-release: app-autoscaler-release-main
+      app-autoscaler-release: app-autoscaler-release-ci
   - task: register-broker
-    file: app-autoscaler-release-main/ci/autoscaler/tasks/register-broker.yml
+    file: app-autoscaler-release-ci/ci/autoscaler/tasks/register-broker.yml
+    input_mapping:
+      release: app-autoscaler-release-ci
```

Comment:
no real point in renaming main to ci tbh

Reply:
I guess if you have another resource ... not seen it in the resources section though.

Reply:
the deploy_autoscaler job is a standalone job that is there for testing purposes at the moment, we use this ci vs main release git resources to be able to tests different pipeline changes without breaking the main flow of test/release.